akka-actor/src/main/mima-filters/2.6.15.backwards.excludes/30389-logging-adapter-notify.excludes

```diff
@@ -0,0 +1,2 @@
+# additional overload added
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.event.LoggingAdapter.notifyWarning")
```

Comment:
I tried it out and it looks like we can delete this exclude now and MiMa is still happy